sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakePathAsyncClient.java

```diff
@@ -59,6 +59,7 @@
     private final ClientLogger logger = new ClientLogger(DataLakePathAsyncClient.class);
 
     final DataLakeStorageClientImpl dataLakeStorage;
+    final DataLakeStorageClientImpl blobDataLakeStorage;
```

Comment:
Can we add a comment here about why we needed to do this so in the future we remember if another feature comes up that's like this?

Reply:
done